.github/workflows/test.yml

```diff
@@ -0,0 +1,57 @@
+name: Validate 'setup-go'
+on: 
+  push:
+  pull_request:
+  schedule:
+    - cron: 0 0 * * *
+
+jobs:
+  default-version:
+    name: Setup default version
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [macos-latest, windows-latest, ubuntu-16.04, ubuntu-18.04]
```

Comment:
I think we should add `ubuntu-20.04`

Reply:
I think `macos-latest, windows-latest, ubuntu-latest` is enough since installation is the same

Reply:
Done